membership-attribute-service/app/models/Features.scala

```diff
@@ -33,6 +33,11 @@ object Features {
   }
 
   val unauthenticated = Features(None, adFree = false, adblockMessage = true, None, None, None)
+
+  def notAMember(attributes: Attributes) = {
+    val adFree = attributes.AdFree.getOrElse(false)
+    Features(None, adFree = adFree, adblockMessage = true, None, None, None)
```

Comment:
not a big deal but you can do `unauthenticated.copy(adFree = adFree)` to save defining out the other fields again

Reply:
Oh right! I'll do that. Thanks.